evap/static/scss/components/_buttons.scss

```diff
@@ -136,6 +136,9 @@ a:not([href]):not(.disabled) {
 }
 
 .btn-switch-evaluation-language {
+    @include media-breakpoint-down(sm) {
+        width: 100%;
+    }
```

Comment:
I think you can remove this part again, it's okay to keep the original width.
It looks a bit weird in a wide sm just before the breakpoint and we don't have it from md onwards anyway and it still looks fine.
I would prefer having it right-aligned though.

sm:
<img width="538" height="453" alt="image" src="https://github.com/user-attachments/assets/cff9cf82-9fa5-418c-9990-ee3281eebc18" />
md:
<img width="532" height="440" alt="image" src="https://github.com/user-attachments/assets/b2e4b0aa-8810-437e-ab31-46670d70538c" />